### ISSUE TITLE
TypeError: Argument 1 ('target') to MutationObserver.observe must be an instance of Node

### ISSUE THREAD

**Issue by andrewhavens** [cibernox/ember-basic-dropdown#423]
I'm trying to implement a basic dropdown:

```hbs
  {{#basic-dropdown as |dropdown|}}
    {{#dropdown.trigger}}Click me{{/dropdown.trigger}}
    {{#dropdown.content}}Content of the trigger{{/dropdown.content}}
  {{/basic-dropdown}}
```

However, I get this error when clicking on the dropdown trigger:
> TypeError: Argument 1 ('target') to MutationObserver.observe must be an instance of Node

Am I doing something wrong? Am I missing a dependency? I'm using the latest version of Ember (3.3.1), ember-islands (1.4.0) and ember-cli-rails.

**Comment by NickBeukema** [comment]
I'm seeing this same issue with my application. Using Ember 3.4 right now. Currently just trying to get the basic "Click Me" demo displayed on the homepage working as a base case.

When the page first loads, I'm seeing the correct DOM for the dropdownElement:

    <div id="ember-basic-dropdown-content-ember263" class="ember-basic-dropdown-content-placeholder" style="display: none;"></div>

Once clicking the Trigger element, the DropdownElement switches from the above to the following:

    <div class="ember-basic-dropdown-content-wormhole-origin">
    </div>

Replacing the content, and the mentioned error is happening because the actual element that's passed to `MutationObserver.observe` is gone.

I've also tried running this on a fresh Ember 3.4 app and it does work as intended, and I've also copied my entire `package.json` file over and just the presence of the packages didn't break it either, so it has something to do with integrating the addon with an existing app, possibly having a conflict with another module. I am using `ember-wormhole` manually in other places, so that could be suspect. I'll continue digging.

Here's a Gist of my current package.json file: https://gist.github.com/NickBeukema/617b5b53c4e57fc9ac3c0b20400c525e
